torch/csrc/distributed/autograd/engine/dist_engine.cpp

```diff
@@ -279,7 +279,7 @@ void DistEngine::computeDependencies(
 
     // Create a dummy GraphRoot and run init_to_execute with it.
     GraphRoot dummyRoot(edges, {});
-    graphTask->init_to_execute(dummyRoot, outputEdges, /*accumulate_grad=*/false);
+    graphTask->init_to_execute(dummyRoot, outputEdges, /*accumulate_grad=*/false, /*min_seq_nr=*/0);
```

Comment:
Setting min_seq_nr=0 means that init_to_execute behavior does not change, i.e., no nodes are skipped